rhel9/common.sh

```diff
@@ -45,3 +45,33 @@ _gdrcopy_enabled() {
     fi
     return 1
 }
+
+# Read a config file and convert newlines to spaces
+_read_conf_file() {
+    local file="$1"
+    [ -f "$file" ] && tr '\n' ' ' < "$file"
+}
+
+# Build driver configuration for state comparison
+_build_driver_config() {
+    cat <<EOF
+DRIVER_VERSION=${DRIVER_VERSION}
+DRIVER_TYPE=${DRIVER_TYPE:-passthrough}
+KERNEL_VERSION=$(uname -r)
+GPU_DIRECT_RDMA_ENABLED=${GPU_DIRECT_RDMA_ENABLED:-false}
+USE_HOST_MOFED=${USE_HOST_MOFED:-false}
+KERNEL_MODULE_TYPE=${KERNEL_MODULE_TYPE:-auto}
```

Comment:
Question -- should we define a default value for this variable (and others) here or just rely on the default defined in `nvidia-driver`?